### ISSUE TITLE
Scroller glitch with one-line long code snippets

### ISSUE THREAD

**Issue by szpak** [zzossig/hugo-theme-zzo#233]
Minor thing, but having one-line long code snippet there is a vertical scroller visualisation glitch when code doesn't need to be scrolled. Example from [your demo](https://themes.gohugo.io//theme/hugo-theme-zzo/en/posts/syntax-highlight/#python) (a grey element under line number):
![image](https://user-images.githubusercontent.com/148013/76645976-625dd500-655a-11ea-8a6c-29e494428313.png)

**Comment by zzossig** [comment]
But this doesn't happen to me,
Are you using Ubuntu?

**Comment by szpak** [comment]
No, I use Fedora. However, I've checked that it doesn't appear in Chromium (79), but in Firefox (74) (and maybe in some other browsers).
